<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtomSelectionTestDialog</class>
 <widget class="QDialog" name="AtomSelectionTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection expression checker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>âœ“</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="topologyEdit">
     <property name="placeholderText">
      <string>topology file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QPlainTextEdit" name="selectionEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>name CA and resi 1 to 2</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Chain</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Res.#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Res.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Atom</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>topologyEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>selectionEdit</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>pressed()</signal>
   <receiver>AtomSelectionTestDialog</receiver>
   <slot>browseTopology()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseTopology()</slot>
 </slots>
</ui>
